packages/core/src/db/models/Labels.ts

```
@@ -0,0 +1,56 @@
+          { $set: { ...doc, userId: user._id } },
+          { new: true }
+        );
+      }
+
+      return models.Labels.create({ ...doc, userId: user._id });
+    }
+
+    public static async removeLabel(name: string) {
+      return models.Labels.findOneAndDelete({ name });
+    }
+  }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)